.eslintrc.js

```diff
@@ -27,7 +28,8 @@ module.exports = {
     "no-param-reassign": 1,
     "prettier/prettier": "error",
     "react/jsx-filename-extension": ["error", { "extensions": [".js", ".jsx", ".ts", ".tsx"] }],
-    '@typescript-eslint/indent': ['error', 2]
+    '@typescript-eslint/indent': ['error', 2],
+    "react/jsx-curly-newline": 0,
```

Comment:
kinda looks like the auto-formatter wasn't run on this, since this line has double quotes and the one before is single quotes